### PR TITLE
Closes #136 Update get_terms function

### LINKED ISSUE #136: Query dataset names being updated in admiral core
See: https://github.com/pharmaverse/admiral/issues/1907

Need to update query data sets created in admiral.test and also example function `get_terms`.

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! 

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. 

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+variables are as follows:
+  
+     - `VAR_PREFIX` to `PREFIX`
+     - `QUERY_NAME` TO `GRPNAME`
+     - `QUERY_ID` TO `GRPID`
+     - `QUERY_SCOPE` TO `SCOPE`
+     - `QUERY_SCOPE_NUM` TO `SCOPEN`
+     - `TERM_LEVEL` TO `SRCVAR`
+     - `TERM_NAME` TO `TERMNAME`
+     - `TERM_ID` TO `TERMID`
```
Comment (bundfussr) [inline NEWS.md:13]: "to" should be lower case.

File: inst/WORDLIST [hunk 2]
```diff
@@ -9,6 +9,7 @@ DM
 DS
 EG
 GlaxoSmithKline
+GRPID
```
Comment (bundfussr) [inline inst/WORDLIST:12]: Do we need this?

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 1 above)
Comment (millerg23) [reply NEWS.md]: Done, thanks.

File: inst/WORDLIST (hunk 2 above)
Comment (millerg23) [reply inst/WORDLIST:12]: Yes its needed, have added it back in.

Comment (bundfussr) [reply inst/WORDLIST:12]: Strange, it seems that the backticks are not processed correctly when the man pages are generated. However, most likely nobody will read the man page. Thus we can ignore it.

**Review by bundfussr — APPROVED** [review]

**Review by zdz2101 — APPROVED** [review]

**Pull request merged**